env.sample

```diff
@@ -1,8 +1,13 @@
+# Superuser for migrations
+POSTGRES_ADMIN_USER=postgres
+POSTGRES_ADMIN_PASSWORD=postgres
+# Minimal user with restricted access
```

Comment:
Can we add a snippet in the readme how multi-tenancy is handled? And also the ways to create a tenant Postgres user.

Let's add a DB Init script in the `docker-compose.yml` file as well

Reply:
updated in commit [here](https://github.com/KeyValueSoftwareSystems/authentication-service/pull/80/commits/1e7837bc70ac59d0cac7c63fbdcdf794815b5251)